ios/FluentUI.Demo/FluentUI.Demo/Demos/TableViewCellDemoController.swift

```diff
@@ -11,7 +11,20 @@ import UIKit
 class TableViewCellDemoController: DemoTableViewController {
     let sections: [TableViewSampleData.Section] = TableViewCellSampleData.sections
 
-    private var isGrouped: Bool = false {
+    override init(nibName nibNameOrNil: String?, bundle nibBundleOrNil: Bundle?) {
```

Comment:
Why do we need to add this if it's already set in DemoTableViewController?

Reply:
this is for the shimmertableviewcell. it isn't using the custom route, it uses the tableviewcelldemo directly.